cylc/flow/parsec/fileparse.py

```diff
@@ -224,6 +228,31 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
     do_empy = True
     do_jinja2 = True
     do_contin = True
+
+    if not template_vars:
+        template_vars = {}
```

Comment:
The cli `template_vars` should be able to override those coming from the plugin(s).

Reply:
This line creates a blank dict if the `template_vars` kwarg isn't given, so I'm not sure why it would prevent kwarg template vars over-riding the plugin. I'll:
- Check whether it's actually necessary
- Check the rest of the code, assuming that this is a general comment.